index.html

```diff
@@ -68,18 +68,34 @@ <h2 id="introduction">Introduction</h2>
 <section id="terminology">
 	<h3>Terminology</h3>
 	<p>This section defines terminology necessary to understand the contents of this document. Most of the terms defined here are specific to this document. Terminology borrowed from other Internationalization documents have a link to the original definition.</p>
+
+<p class="definition"><dfn data-lt="metadata|metadata">Metadata</dfn> is information about data defined in terms of functions, form and scope. In this document, the function of metadata is to express information about direction and language. The form for direction metadata is described in [[[#bidi-approaches]]], the form for language metadata is described in [[[#language-approaches]]]. In this document, the scope for both types of metadata is a string or a set of strings. In absence of direction or language metadata, defaults apply, see [[[#resource_wide_defaults]]].</p>
```

Comment:
i think there should be a comma after 'information about data', otherwise it's slightly ambiguous.

Reply:
... although i'm not actually clear what is meant by "defined in terms of functions, form and scope", anyway.

Reply:
The sentence about 'in this document..' is a good addition. But the following sentences worry me.

Rather than 'The form for xxx metadata is described in'  i think you mean 'Various possible ways of representing xxx metadata are described in...'. (Note that not all are recommended.) 

But actually, i'm not convinced that we need those two links, anyway.  If you're going to point to them, then why not link to the 2 sections that contain our actual recommendations, too.  But if you do that, you're largely just repeating what's clear anyway in the table of contents.

Finally, 'In absence of' -> 'In the absence of'.

Reply:
This was Felix's pull request verbatim. I have removed the bottom linked part of the paragraph and rewritten the definition. See what you think.